Screen-Sharing-Video-Chat /js/app.js

```diff
@@ -0,0 +1,123 @@
+/* global OT API_KEY TOKEN SESSION_ID SAMPLE_SERVER_BASE_URL */
+
+let apiKey;
+let sessionId;
+let token;
+let screenPublisher;
+let session;
+let publisher;
+let canPublish = false;
+// Handling all of our errors here by alerting them
+function handleError(error) {
+  if (error) {
+    alert(error.message);
+  }
+}
+
+
+function toggleScreen() {
+  const screenSharing = document.getElementById('screen-sharing');
+  // If the screen publisher is connected to the session, unpublish screen and re-publish camera footage
+  if (screenPublisher.session) {
+    session.unpublish(screenPublisher);
+    session.publish(publisher, handleError);
+    screenSharing.classList.remove('toggle-button-on');
+    screenSharing.classList.add('toggle-button-off');
+  // Else, if the camera publisher is connected to the session, unpublish camera footage and re-publish screen
+  } else if (publisher.session) {
+    session.unpublish(publisher);
+    if (canPublish) {
+      session.publish(screenPublisher, handleError);
+    }
+    screenSharing.classList.add('toggle-button-on');
+    screenSharing.classList.remove('toggle-button-off');
+  }
+}
+
+function preventDefaults() {
+  // If a stream is destroyed BY THIS CLIENT's publishers override default behaviour.
+  publisher.on('streamDestroyed', function preventSessionDefault(event) {
+    if (event.stream.connection.connectionId === session.connection.connectionId) {
+      event.preventDefault();
+    }
+  });
+
+  screenPublisher.on('streamDestroyed', function preventPublisherDefault(event) {
+    if (event.stream.connection.connectionId === session.connection.connectionId) {
+      event.preventDefault();
+    }
+  });
+}
+
+function initializeSession() {
+  session = OT.initSession(apiKey, sessionId);
+
+  // Subscribe to a newly created stream
+  session.on('streamCreated', function streamCreated(event) {
```

Comment:
This is not checking the `videoType` property of the stream. I think we should put the screen-sharing stream in the large container, and put the camera stream in the small container.